testing/scenario_app/ios/Scenarios/ScenariosTests/FlutterViewControllerTest.m

```diff
@@ -0,0 +1,60 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#import <Flutter/Flutter.h>
+#import <XCTest/XCTest.h>
+#import "AppDelegate.h"
+
+static NSBundle* FindTestBundle() {
+  for (NSBundle* bundle in [NSBundle allBundles]) {
+    if ([bundle.bundlePath containsString:@".xctext"]) {
```

Comment:
Should this be xctest?  If this test is passing, does that prove this isn't needed (because there are no . xctext bundles)?